notebooks/write_CassiniIssPds3LabelNaifSpiceDriver.ipynb

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
I think most of these notebooks were scrubbed from ALE as they contain full paths. We either need to scrub them from this PR again or change the paths